analyses/immune-deconv/README.md

```diff
@@ -0,0 +1,64 @@
+## Immune Deconvolution
```

Comment:
Discussed in person: Use one sentence per line same as manubot

Reply:
Changed to one sentence per line: https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/292/commits/10485cfeb6de79267be7331d26d93b96cc8e369c